[libbacktrace] update to fix build error on x64-linux

### Is your feature request related to a problem? Please describe.

I'm encountering an issue while trying to install boost:x64-linux using vcpkg. The installation fails during the build of libbacktrace:x64-linux, specifically at the step involving autoreconf. Despite running git pull and vcpkg update to ensure the latest port files are used, the error persists.

### Proposed solution

Please update the libbacktrace port or make any necessary changes to fix the build failure when installing boost:x64-linux. This issue is specifically related to the autoreconf command that fails during the build process.

### Describe alternatives you've considered

I have tried executing git pull and vcpkg update to ensure I'm working with the latest files, but the issue still occurs. No alternative solutions have worked so far.

### Additional context

Computing installation plan...
The following packages will be built and installed:
    boost:x64-linux@1.86.0#1
  * boost-stacktrace[backtrace,core]:x64-linux@1.86.0
  * boost-statechart:x64-linux@1.86.0
  * boost-stl-interfaces:x64-linux@1.86.0
  * boost-test:x64-linux@1.86.0
  * boost-timer:x64-linux@1.86.0
  * boost-url:x64-linux@1.86.0
  * boost-wave:x64-linux@1.86.0
  * boost-yap:x64-linux@1.86.0
  * libbacktrace:x64-linux@2023-11-30#1
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-linux...
Compiler found: /usr/bin/c++
Restored 0 package(s) from /root/.cache/vcpkg/archives in 1.46 ms. Use --debug to see more details.
Installing 1/10 libbacktrace:x64-linux@2023-11-30#1...
Building libbacktrace:x64-linux@2023-11-30#1...
-- Using cached ianlancetaylor-libbacktrace-14818b7783eeb9a56c3f0fca78cefd3143f8c5f6.tar.gz.
-- Cleaning sources at /vcpkg/buildtrees/libbacktrace/src/3143f8c5f6-8b4b375662.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /vcpkg/downloads/ianlancetaylor-libbacktrace-14818b7783eeb9a56c3f0fca78cefd3143f8c5f6.tar.gz
-- Using source at /vcpkg/buildtrees/libbacktrace/src/3143f8c5f6-8b4b375662.clean
-- Getting CMake variables for x64-linux-dbg
-- Getting CMake variables for x64-linux-rel
-- Generating configure for x64-linux
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: /usr/bin/autoreconf -vfi
    Working Directory: /vcpkg/buildtrees/libbacktrace/src/3143f8c5f6-8b4b375662.clean/
    Error code: 2
    See logs for more information:
      /vcpkg/buildtrees/libbacktrace/autoconf-x64-linux-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:731 (vcpkg_execute_required_process)
  ports/libbacktrace/portfile.cmake:12 (vcpkg_configure_make)
  scripts/ports.cmake:196 (include)


error: building libbacktrace:x64-linux failed with: BUILD_FAILED
See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
Elapsed time to handle libbacktrace:x64-linux: 2.6 s
Please ensure you're using the latest port files with git pull and vcpkg update.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+libbacktrace
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=[libbacktrace]+Build+error+on+x64-linux&body=Copy+issue+body+from+%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md

--- a/ports/libbacktrace/portfile.cmake
+++ b/ports/libbacktrace/portfile.cmake
@@ -5,8 +5,8 @@ endif()
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO ianlancetaylor/libbacktrace
-    REF 14818b7783eeb9a56c3f0fca78cefd3143f8c5f6
-    SHA512 d96c337cda6d230b162d983b2ab6ff6643895158f1d6f2e814bf28a2212a0cf46313935c2ed95a4408e6ad1da3c0c1ccb09847cf8b8b2ca6ad299101b8f79dd4
+    REF 1db85642e3fca189cf4e076f840a45d6934b2456
+    SHA512 a7f7a1233f551326e4ae1ba91db0fb905cf2737c20284c9aaf26cfe448b2a54efeaaa678e3abccbe0856c2a19019412208da7c1a82d319a58fe4d66d0a952aa0
 )
 
 vcpkg_configure_make(

--- a/ports/libbacktrace/vcpkg.json
+++ b/ports/libbacktrace/vcpkg.json
@@ -1,7 +1,6 @@
 {
   "name": "libbacktrace",
-  "version-date": "2023-11-30",
-  "port-version": 1,
+  "version-date": "2024-11-30",
   "description": "The libbacktrace library may be linked into a program or library and used to produce symbolic backtraces.",
   "homepage": "https://github.com/ianlancetaylor/libbacktrace",
   "license": "BSD-3-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4349,8 +4349,8 @@
       "port-version": 7
     },
     "libbacktrace": {
-      "baseline": "2023-11-30",
-      "port-version": 1
+      "baseline": "2024-11-30",
+      "port-version": 0
     },
     "libbf": {
       "baseline": "1.0.0",

--- a/versions/l-/libbacktrace.json
+++ b/versions/l-/libbacktrace.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "c231fe43ac03b2250f605fac80e2cc80eb025bb8",
+      "version-date": "2024-11-30",
+      "port-version": 0
+    },
+    {
       "git-tree": "b3758864f5b04051a1db75cd9842a05f4afcdcb5",
       "version-date": "2023-11-30",
       "port-version": 1

